.drone.star

```diff
@@ -622,26 +622,39 @@ def website(ctx):
     },
     'steps': [
       {
-        'name': 'generate',
+        'name': 'prepare',
+        'image': 'owncloudci/alpine:latest',
+        'commands': [
+          'apk add make',
```

Comment:
this is redundant, the image contains `make`:

```console
docker run -it owncloudci/alpine make
make: *** No targets specified and no makefile found.  Stop.
```

Reply:
You're right. Forgot to remove it.

Reply:
@refs Removed